clustering/common/src/main/java/org/jboss/as/clustering/controller/ReloadRequiredWriteAttributeHandler.java

```diff
@@ -41,4 +47,15 @@ public ReloadRequiredWriteAttributeHandler(WriteAttributeStepHandlerDescriptor d
     public void register(ManagementResourceRegistration registration) {
         this.descriptor.getAttributes().forEach(attribute -> registration.registerReadWriteAttribute(attribute, null, this));
     }
+
+    @Override
+    protected void finishModelStage(OperationContext context, ModelNode operation, String attributeName, ModelNode newValue, ModelNode oldValue, Resource model) throws OperationFailedException {
+        super.finishModelStage(context, operation, attributeName, newValue, oldValue, model);
+
+        if (!context.isBooting()) {
+            TransformerOperationAttachment attachment = TransformerOperationAttachment.getOrCreate(context);
+            attachment.attachIfAbsent(PreviousAttributeValueOperationContextAttachment.KEY, new PreviousAttributeValueOperationContextAttachment(oldValue.clone()));
```

Comment:
Good catch, should be addressed by org.jboss.as.clustering.controller.transform.InitialAttributeValueOperationContextAttachment implementation now.